src/simulators/statevector/qubitvector.hpp

```diff
@@ -539,6 +525,10 @@ class QubitVector {
   // Swap cols and rows of vectorized matrix
   void swap_cols_and_rows(const uint_t idx1, const uint_t idx2,
                           cvector_t &mat, uint_t dim) const;
+
+  cvector_t expand_matrix(const reg_t& src_qubits,
```

Comment:
Can you add a description of what this function does and what the arguments should be.
I think its something like `src_qubits` are the qubits for the matrix, and `dst_sorted_qubits` are the sorted list of qubits in the output space it will be expanded to